examples/ws-fasthttp-reverse-proxy-dynamic/ws_proxy.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/fasthttp/websocket"
+	"github.com/valyala/fasthttp"
+
+	proxy "github.com/yeqown/fasthttp-reverse-proxy/v2"
+)
+
+var (
+	proxyServer *proxy.WSReverseProxy
+	once        sync.Once
+	mainServer  = fmt.Sprintf("ws://localhost:8080")
+)
+
+// ProxyHandler ... fasthttp.RequestHandler func
+func ProxyHandler(ctx *fasthttp.RequestCtx) {
+	once.Do(func() {
+		var err error
+		proxyServer, err = proxy.NewWSReverseProxyWith(
+			proxy.WithURL_OptionWS(mainServer),
+			proxy.WithUpgrader_OptionWS(&websocket.FastHTTPUpgrader{
+				CheckOrigin: func(r *fasthttp.RequestCtx) bool { return true },
+			}),
```

Comment:
Add feature enabled Option here explicitly  if you want to use the dynamic host feature.

Reply:
Added explicit way to enable dynamic path overriding option: `WithDynamicPath_OptionWS`